NewSpring/ApollosProject
# Added Delete Icon to User Prayer List

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR adds a delete icon and uses that icon in place of the "close" icon so that deleting a prayer looks oh so much better.

![Screenshot 2019-07-18 15 42 51](https://user-images.githubusercontent.com/3229463/61487483-02377480-a974-11e9-9a37-4766e6557c52.png)

### How do I test this PR?
In your sim, go to your prayers and see the beauty of the delete icon. It's functional too!

---
> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
